roundedimageview/src/main/java/com/makeramen/roundedimageview/RoundedDrawable.java

```diff
@@ -101,9 +102,16 @@ public static Drawable fromDrawable(Drawable drawable) {
         // loop through layers to and change to RoundedDrawables if possible
         for (int i = 0; i < num; i++) {
           Drawable d = ld.getDrawable(i);
+          if(ld.getId(i) <= 0) {
+            // workaround: setId to make LayerDrawable works
+            ld.setId(i, i+1);
```

Comment:
Hm, this looks scary, are you sure this doesn't have other consequences? It's a neat hack, but I would like to make this as safe as possible and make sure we're not conflicting with our use of IDs.